executor_manager/common/distributed_lock.py

```
@@ -0,0 +1,104 @@
+            lock_name: Name of the lock to release
+
+        Returns:
+            True if released successfully, False otherwise
+        """
+        if self.redis_client is None:
+            return False
+
+        lock_key = f"{LOCK_KEY_PREFIX}{lock_name}"
+        try:
+            self.redis_client.delete(lock_key)
+            return True
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)